build.gradle

```diff
@@ -445,31 +449,34 @@ def fineTuneEclipseClasspathFile(eclipse, project) {
   }
 }
 
-// Aggregates all jacoco results into the root project directory
-task jacocoRootReport(type: org.gradle.testing.jacoco.tasks.JacocoReport) {
-  def javaProjects = subprojects.findAll { it.path != ':core' }
 
-  description = 'Generates an aggregate report from all subprojects'
-  dependsOn(javaProjects.test)
+if (!JavaVersion.current().isJava10Compatible()) {
```

Comment:
Maybe a comment linking to https://github.com/jacoco/jacoco/issues/629 would help make it clear why this is here?